pkg/server/container_create_test.go

```diff
@@ -146,6 +147,9 @@ func getCreateContainerTestData() (*runtime.ContainerConfig, *runtime.PodSandbox
 		assert.Contains(t, spec.Process.User.AdditionalGids, uint32(1111))
 		assert.Contains(t, spec.Process.User.AdditionalGids, uint32(2222))
 
+		t.Logf("Check NoNewPrivs")
```

Comment:
nit: Check no_new_privs.

This is linux term, and also lower case which is consistent with other logs.

Just a small nit. :)